src/Goerli-DssSpell.t.base.sol

```diff
@@ -296,13 +296,13 @@ contract GoerliDssSpellTestBase is Config, Test, DSMath {
 
     function readInput(string memory input) internal returns (string memory) {
         string memory root = vm.projectRoot();
-        string memory path = stringConcat(stringConcat(root, "/config/"), stringConcat(input, ".json"));
+        string memory path = stringConcat(stringConcat(root, "/scripts/input/5/"), stringConcat(input, ".json"));
         return vm.readFile(path);
     }
```

Comment:
Can you use this code?

```
function readInput(string memory input) internal returns (string memory) {
	string memory root = vm.projectRoot();
	string memory chainInputFolder = string.concat("/script/input/", vm.toString(block.chainid), "/");
	return vm.readFile(string.concat(root, chainInputFolder, string.concat(input, ".json")));
}
```

Reply:
yeah so unfortunately both `block.chainId` and `string.concat` are a solidity 0.8 things. so the 0.6.12 code needs to become:

```solidity
function getChainID() external view returns (uint256) {
    uint256 id;
    assembly {
        id := chainid()
    }
    return id;
}

function readInput(string memory input) internal returns (string memory) {
    string memory root = vm.projectRoot();
    string memory chainInputFolder = stringConcat(stringConcat("/scripts/input/", vm.toString(getChainId())),"/");
    string memory path = stringConcat(stringConcat(root, chainInputFolder), stringConcat(input, ".json"));
    return vm.readFile(path);
}
```

I'm not sure that is worth it to avoid having to hardcode 5 for goerli and 1 for mainnet.  Do you think it is worth that?

For "when we update', we're going to set a meeting on this, but from the planning call, it sounded like it might not have support.  I think it requires a lot of effort and there are still optimizer bugs being discovered in 0.8.x.  One possible option would be to move multichain stuff to its own repo and import that into spells and xdomain, but then this code would still be 0.6.12 for spells

Reply:
Sure. Not a big deal if you want to leave it like this I guess.